### DISCUSSION TITLE
Welcome to Hamlib Discussions!

### DISCUSSION THREAD

**Discussion by mdblack98** [Hamlib/Hamlib#1546] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DavidTMenges** [comment]
I am WB9TEN and an Extra. Currently I am working on Vara, Flrig, Hamlib and trying to write some Bash scripts and Windows batch files. I maintain a BPQ node that will soon have a packet RMS followed by a Vara HF RMS running on a Raspberry Pi 4.

**Comment by steve1427** [comment]
Hi, I'm Steve VK3SPX in Baringhup, Victoria, Australia. I've developed some Windows Python programs to assist with Hamlib. The main program allows selection of the radio type and executes rigctld in the background so other multiple programs can access it. Thus includes the likes of LOG4OM, WSJT-X etc, but also some of my own Python rigctld client programs that for example scan various frequencies for activity and other things. I do have a specific problem with using Hamlib with a Flex radio, but I'll post that in another place.

**Comment by Haig123** [comment]
Hello, my name is Tim.  Ive use hamlib mostly for home constructed az/el rotators used for ham satellite, satnogs station, weather sats and now EME.  

Hoping there is a solution to my current issue , which I’ll start in a new discussion.
